### PR TITLE
Separating e2e test install scripts

### PR DESCRIPTION
**Pull request by dimpar**
We do not need dApps install scripts to run e2e nightly test.
Conditional execution should reduce the run time and another point of failure.

### CODE REVIEW

**Comment by nkuba** [comment]
Thoughts on handling this in a boolean env variable?

**Comment by dimpar** [comment]
Sure, did you have in mind something like in fa16e0f ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
           command: cd ~ && ./local-setup/deployments/local-setup-instance/provisioning/run-bitcoin.sh
       - run:
```
Comment (nkuba) [inline .circleci/config.yml:62]: If we're in this land could we try to improve the steps declaration and replace all
`cd ~ && ./local-setup/deployments/local-setup-environment/provisioning` with `working_directory` property, e.g.:
```yaml
      - run:
          name: Install common tools
          working_directory: ~/local-setup/deployments/local-setup-environment/provisioning
          command: ./install-commons.sh
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ jobs:
...
           name: Run install script
-          command: cd ~ && ./local-setup/deployments/local-setup-instance/provisioning/run-install.sh
+          command: cd ~ && export E2E_TEST=true && ./local-setup/deployments/local-setup-instance/provisioning/run-install.sh
```
[the remark this answers is not in the document]
Comment (dimpar) [reply .circleci/config.yml]: Done.

File: .circleci/config.yml (hunk 1 above)
Comment (dimpar) [reply .circleci/config.yml:67]: Done.

**Review by nkuba — APPROVED** [review]

**Pull request merged**